tuning/test_tune.py

```diff
@@ -344,4 +344,355 @@ def test_apply_params_batch_matmul():
     )
     assert "tile_sizes = [[1, 416, 320, 128]]" in modified
     assert "waves_per_eu = 1 : i64" in modified
-    
\ No newline at end of file
+
+def test_parse_mlir():
+    mlir_str = r"""
+    builtin.module  {
+      func.func @simple_mul(%arg0: tensor<4xf32>, %arg1: tensor<4xf32>) -> tensor<4xf32> {
+        %0 = arith.mulf %arg0, %arg1 : tensor<4xf32>
+        return %0 : tensor<4xf32>
+      }
+    }
+  """
+    mlir_module = tune.parse_mlir(mlir_str)
+    assert mlir_module != None
+    assert isinstance(mlir_module, tune.ireec._mlir_libs._mlir.ir.Module) == True
+    assert isinstance(mlir_module.body.operations[0], tune.ireec.dialects.func.FuncOp) == True
```

Comment:
```suggestion
    assert isinstance(mlir_module, tune.ireec._mlir_libs._mlir.ir.Module)
    assert isinstance(mlir_module.body.operations[0], tune.ireec.dialects.func.FuncOp)
```

These are already booleans

Reply:
Not resolved yet. Also below.